### ISSUE TITLE
DFC: Update File Repository Page for new token manager

### ISSUE THREAD

**Issue by rosibaj** [kids-first/kf-portal-ui#1238]
- [x] page needs to be aware of BOTH fence tokens.  Update to use Jons State Provider that he built already.  
- [x] Update last column of table with this logic:
- If the user does NOT have access to the file show a lock (ask Kim for the new teal svg)
- If the user DOES have access to the file and it is a target file, show the disabled download icon with the tooltop `This file is not available for download. Please use Cavatica to download` AND and active Cavatica Icon (ask Kim for the new teal svg)
- If the DOES have access to the file and it is a NOT a target file, show the enabled download icon with the current behaviour of downloading when clicked; also show an active active Cavatica Icon
- [x] When the Cavatica icon in the action column is clicked, it launched the Push to Cavatica flow for that single file

Zeplin: https://zpl.io/VOPkJP8

![screen shot 2019-02-21 at 4 36 21 pm](https://user-images.githubusercontent.com/18424138/53203328-d8616800-35f6-11e9-949c-a8a28913f1bd.png)

**Comment by rosibaj** [comment]
@joneubank yes; I can move that to another ticket for St Justine though if you want?

**Comment by joneubank** [comment]
I don't mind doing it.

**Comment by rosibaj** [comment]
blocked on testing bc DCF staging says "unsecure login"
<img width="688" alt="Screen Shot 2019-04-03 at 5 37 52 PM" src="https://user-images.githubusercontent.com/15824252/55514995-6ccbdb00-5637-11e9-89fc-2c4d68be2a88.png">

**Comment by joneubank** [comment]
Let CHOP know right away

**Comment by rosibaj** [comment]
yup alerted on slack!
